### PR TITLE
Tests/repository tests

### PR DESCRIPTION
**Pull request by imbieras**

### CODE REVIEW

File: StudyBuddy/Controllers/ProfileController/ProfileController.cs [hunk 1]
```diff
@@ -139,7 +139,9 @@ public async Task<IActionResult> SaveProfile(ProfileDto profileDto)
                 traits.Latitude = parsedLatitude;
             }
 
-            await _userService.RegisterUserAsync(profileDto.Name, flags, traits, profileDto.Hobbies);
+            UserId uid = await _userService.RegisterUserAsync(profileDto.Name, flags, traits, profileDto.Hobbies);
+
+            await _userService.BanUserAsync(uid);
```
Comment (imbieras) [inline StudyBuddy/Controllers/ProfileController/ProfileController.cs:144]: ????????

File: StudyBuddy/Data/ApplicationDbContext.cs [hunk 2]
```diff
@@ -25,13 +24,14 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
         modelBuilder.UseValueConverterForType(typeof(Guid), GuidConverter.GuidToStringConverter);
+        modelBuilder.UseValueConverterForType(typeof(List<string>), StringListConverter.StringListToStringConverter);
 
         // Configure Users
         modelBuilder.Entity<User>().OwnsOne(u => u.Traits);
 
-        modelBuilder.Entity<User>()
-            .Property(u => u.HobbiesArray)
-            .HasColumnName("HobbiesArray");
+        // modelBuilder.Entity<User>()
```
Comment (imbieras) [inline StudyBuddy/Data/ApplicationDbContext.cs:32]: Why is this still here?

**Review by imbieras — COMMENTED** [review]
Some weird things left

**Review by canyonhhh — APPROVED** [review]
Looks like these tests have passed not only the code but also the vibe check. Merge approved! 👍

**Pull request merged**